src/packs/nzism-v36-1022-20.ts

```diff
@@ -0,0 +1,945 @@
+  ELBLoggingEnabled,
+  ELBTlsHttpsListenersOnly,
+} from '../rules/elb';
+import { IAMPolicyNoStatementsWithAdminAccess } from '../rules/iam';
+import { KMSBackingKeyRotationEnabled } from '../rules/kms';
+import { LambdaFunctionPublicAccessProhibited } from '../rules/lambda';
+import {
+  OpenSearchEncryptedAtRest,
+  OpenSearchInVPCOnly,
+  OpenSearchNodeToNodeEncryption,
+} from '../rules/opensearch';
+import {
+  RDSAutomaticMinorVersionUpgradeEnabled,
+  RDSInBackupPlan,
+  RDSInstanceDeletionProtectionEnabled,
+  RDSInstancePublicAccess,
+  RDSLoggingEnabled,
+  RDSMultiAZSupport,
+  RDSStorageEncrypted,
+} from '../rules/rds';
+import {
+  RedshiftRequireTlsSSL,
+  RedshiftBackupEnabled,
+  RedshiftClusterConfiguration,
+  RedshiftClusterMaintenanceSettings,
+  RedshiftClusterPublicAccess,
+} from '../rules/redshift';
+import {
+  S3BucketLevelPublicAccessProhibited,
+  S3BucketLoggingEnabled,
+  S3BucketPublicReadProhibited,
+  S3BucketPublicWriteProhibited,
+  S3BucketServerSideEncryptionEnabled,
+  S3BucketSSLRequestsOnly,
+  S3BucketVersioningEnabled,
+  S3DefaultEncryptionKMS,
+} from '../rules/s3';
+import {
+  SageMakerEndpointConfigurationKMSKeyConfigured,
+  SageMakerNotebookInstanceKMSKeyConfigured,
+  SageMakerNotebookNoDirectInternetAccess,
+} from '../rules/sagemaker';
+import { SecretsManagerUsingKMSKey } from '../rules/secretsmanager';
+import { SNSEncryptedKMS } from '../rules/sns';
+import {
+  VPCFlowLogsEnabled,
+  VPCDefaultSecurityGroupClosed,
+} from '../rules/vpc';
+import { WAFv2LoggingEnabled } from '../rules/waf';
+
+/**
+ * Check for NZISM v36-1022-20 compliance.
+ * Based on the NZISM v36-1022-20: <URL To be updated once published>
+ * */
+
+export class NZISM36Checks extends NagPack {
+  constructor(props?: NagPackProps) {
+    super(props);
+    this.packName = 'NZISM3.6';
+  }
+  public visit(node: IConstruct): void {
+    if (node instanceof CfnResource) {
+      this.checkELB(node);
+      this.checkAPIGW(node);
+      this.checkCloudTrail(node);
+      this.checkCloudFront(node);
+      this.checkKMS(node);
+      this.checkCloudWatch(node);
+      this.checkRDS(node);
+      this.checkECS(node);
+      this.checkEFS(node);
+      this.checkDMS(node);
+      this.checkDynamoDB(node);
+      this.checkEC2(node);
+      this.checkElasticBeanstalk(node);
+      this.checkElastiCache(node);
+      this.checkLambda(node);
+      this.checkOpenSearch(node);
+      this.checkIAM(node);
+      this.checkRedshift(node);
+      this.checkS3(node);
+      this.checkSageMaker(node);
+      this.checkSecretsManager(node);
+      this.checkSNS(node);
+      this.checkVPC(node);
+      this.checkWAF(node);
+    }
+  }
+
+  /**
+   * Check API Gateway Resources
+   * E
+   * @param node the CfnResource to check
+   * @param ignores list of ignores for the resource
+   */
+  private checkAPIGW(node: CfnResource): void {
+    this.applyRule({
+      ruleSuffixOverride: 'API_GATEWAY_LOGGING',
```

Comment:
What are these overrides based on? If the overrides are only from the [source identifiers](https://github.com/awslabs/aws-config-rules/blob/master/aws-config-conformance-packs/Operational-Best-Practices-for-NZISM.yaml#L141C17-L141C17) on the conformance pack sample template, we shouldn't use them and just use the default `cdk-nag` rule names. The source identifiers are only there because they map to specific managed config rules. That doesn't matter for the case of `cdk-nag`

Reply:
Ok, i'll remove them.